hera_mc/cm_active.py

```diff
@@ -195,7 +201,8 @@ def load_geo(self, at_date=None):
         at_date = cm_utils.get_astropytime(at_date)
         gps_time = self.set_times(at_date)
         self.geo = {}
-        for ageo in self.session.query(geo_location.GeoLocation).filter(geo_location.GeoLocation.created_gpstime <= gps_time):
+        for ageo in self.session.query(geo_location.GeoLocation).filter(
+                geo_location.GeoLocation.created_gpstime <= gps_time):
```

Comment:
this one is not as bad, but up to your discretion.

Reply:
There appears to be no indent that flake8 will accept for the closing bracket on the next line for this one.